Protos/step_by_step.proto

```diff
@@ -19,16 +22,30 @@ service StepByStepService {
 
     // Checks whether a instruction is implemented correctly.
     rpc CheckInstruction (CheckInstructionRequest) returns (CheckInstructionResponse);
+
+    // Sets the filepath of every instruction to the newly created .cs file.
+    rpc SetNewFilePath (SetFilePathRequest) returns (SetFilePathResponse);
```

Comment:
I'd probably make this a property on the `CheckInstruction` request. This saves a roundtrip to the background service.

Reply:
This has been moved to the logic responsible for making the CheckInstruction.